x-pack/plugins/apm/public/components/shared/Stacktrace/Variables.tsx

```diff
@@ -27,49 +27,34 @@ const VariablesContainer = styled.div`
   font-family: ${fontFamily};
 `;
 
-const VariablesToggle = styled.a`
-  display: block;
-  cursor: pointer;
-  user-select: none;
-`;
-
-const VariablesTableContainer = styled.div`
-  padding: ${px(units.plus)} ${px(unit)} 0;
-`;
-
 interface Props {
   vars: IStackframe['vars'];
 }
 
 export class Variables extends React.Component<Props> {
-  public state = {
-    isVisible: false
-  };
-
-  public onClick = () => {
-    this.setState(() => ({ isVisible: !this.state.isVisible }));
-  };
-
   public render() {
     if (!this.props.vars) {
       return null;
     }
 
     return (
-      <VariablesContainer>
-        <VariablesToggle onClick={this.onClick}>
-          <Ellipsis horizontal={this.state.isVisible} />{' '}
-          {i18n.translate(
-            'xpack.apm.stacktraceTab.localVariablesToogleButtonLabel',
-            { defaultMessage: 'Local variables' }
-          )}
-        </VariablesToggle>
-        {this.state.isVisible && (
-          <VariablesTableContainer>
-            <DottedKeyValueTable data={this.props.vars} maxDepth={5} />
-          </VariablesTableContainer>
-        )}
-      </VariablesContainer>
+      <React.Fragment>
+        <EuiHorizontalRule margin="none" />
```

Comment:
I don't see this making any difference except it adds another border to the panel. I reckon we can remove the `EuiHorizontalRule` here.

Reply:
Horizontal Rule removed